ipywidgets/static/widgets/js/manager.js

```diff
@@ -332,19 +348,19 @@ define([
          *      widget_class: (optional) string
          *          Target name of the widget in the back-end.
          *      comm: (optional) Comm
+         *           Comm object associated with the widget.
+         *      model_id: (optional) string
+         *           If not provided, the comm id is used.
          *
-         * Create a comm if it wasn't provided.
+         * Either a comm or a model_id must be provided.
```

Comment:
I'm not sure that it makes sense to require a comm or model id.  Thinking symmetrically, you don't need to specify a comm or model id when instantiating a widget from the backend.

If not in this method, it makes sense to have a method that can do this, for the case when the user just wants to make a new widget, and doesn't care about comms or model ids.

I do not object to doing this in another PR.